js/status.js

```diff
@@ -0,0 +1,234 @@
+(function ($, window)
+{
+  // register namespace
+  $.extend(true, window, {
+    "StatusApp": StatusApp
+    }
+  );
+
+  /**
+   * This application builds and displays a status table constructed from
+   * queries to a list of services. Each row in the table contains
+   * the status of a service.
+   *
+   * param: _options - values to be supplied to this application
+   */
+  function StatusApp(_options)
+  {
+    const VERSION_PARAM = "?version=" + new Date().getTime();
+    const XML_CONTENT_TYPE = 'application/xml; charset=utf-8';
+    const TEXT_CONTENT_TYPE = 'text/plain; charset=utf-8';
+    const STANDARD_ID = '[standardID="ivo://ivoa.net/std/VOSI#availability"]';
+    const CAPS_SERVERS = ["www.canfar.phys.uvic.ca", "www.cadc-ccda.hia-iha.nrc-cnrc.gc.ca"];
+    const REFRESH_PERIOD = 180000; // 180 seconds or 3 minutes
+    const IMAGES_DIR = _options.images;
+    const RESOURCE_CAPS_URL = _options.resourceCaps;
+
+    // serviceKVPs contains key/value pairs which map service capabilities URL to 
+    // an html element containing a row of status info for a service, e.g.
+    // key=http://canfar.phys.uvic.ca/ac/capabilities
+    // value=<tr class="ac">
+    //         <td class="ac-service">
+    //           ac          
+    //           <img id="load-ac" src="../../images/progress_small.gif"></td>
+    //         <td class="ac-available">true</td>
+    //         <td class="ac-message">/ac is available</td>
+    //         <td class="ac-time">Fri, 18 Aug 2017 14:59:06 GMT</td>
+    //       </tr>
+    var serviceKVPs = {};
+
+    /**
+     * A function to create a node which represents a table cell.
+     *
+     * param: tagName - tag name for the html element in this node
+     * param: className - class name to identify the html element in this ndoe
+     * param: data - the text data associated with this html element
+     */
+    var _createNode = function (tagName, className, data) {
+        var result = document.createElement(tagName);
+
+        result.appendChild(document.createTextNode("" + data));
+        if (className != "")
```

Comment:
Always use `!==` and `===` unless there isa  good reason not to.  This applies to the entire file.